places/pickerington-oh.md

```diff
@@ -0,0 +1,5 @@
+## Pickerington, OH
+
+### Great places to eat
+
+### Fun things to do
```

Comment:
what's fun to do in Pickerington.  I'm actually only in Lewis Center, practically neighbors.